utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -103,83 +105,160 @@ int main(int argc, char **argv)
 
     if (verbosity == 1)
     {
-        printf("\nInputs for this test case are:");
-        printf("\nsrc1 = %s", argv[1]);
-        printf("\nsrc2 = %s", argv[2]);
-        printf("\ndst = %s", argv[3]);
-        printf("\nu8 / f16 / f32 / u8->f16 / u8->f32 / i8 / u8->i8 (0/1/2/3/4/5/6) = %s", argv[4]);
-        printf("\noutputFormatToggle (pkd->pkd = 0 / pkd->pln = 1) = %s", argv[5]);
-        printf("\ncase number (0:86) = %s", argv[6]);
+        if (test_type == 0)
+        {
+            printf("\nInputs for this test case are:");
+            printf("\nsrc1 = %s", argv[1]);
+            printf("\nsrc2 = %s", argv[2]);
+            printf("\ndst = %s", argv[3]);
+            printf("\nu8 / f16 / f32 / u8->f16 / u8->f32 / i8 / u8->i8 (0/1/2/3/4/5/6) = %s", argv[4]);
+            printf("\noutputFormatToggle (pkd->pkd = 0 / pkd->pln = 1) = %s", argv[5]);
+            printf("\ncase number (0:86) = %s", argv[6]);
+            printf("\nUNIT/PERFORMANCE - 0/1 = %s", argv[8]);
+        }
+        else
+        {
+            printf("\nInputs for this test case are:");
+            printf("\nsrc1 = %s", argv[1]);
+            printf("\nsrc2 = %s", argv[2]);
+            printf("\nu8 / f16 / f32 / u8->f16 / u8->f32 / i8 / u8->i8 (0/1/2/3/4/5/6) = %s", argv[4]);
+            printf("\noutputFormatToggle (pkd->pkd = 0 / pkd->pln = 1) = %s", argv[5]);
+            printf("\ncase number (0:86) = %s", argv[6]);
+            printf("\nNumber of times to run = %s", argv[7]);
+            printf("\nUNIT/PERFORMANCE - 0/1 = %s", argv[8]);
+        }
     }
 
-    int ip_channel = 3;
+    Rpp32u elementsInRow;
+    // int layout_switch;
+    clock_t start, end;
+    double start_omp, end_omp;
+    double cpu_time_used, omp_time_used;
+
+    if (argc < MIN_ARG_COUNT)
+    {
+        printf("\nImproper Usage! Needs all arguments!\n");
+        if (test_type == 0)
+        {
+            if (layout_switch == 0)
+                printf("\nUsage: ./Tensor_host_pkd3 <src1 folder> <src2 folder (place same as src1 folder for single image functionalities)> <u8 = 0 / f16 = 1 / f32 = 2 / u8->f16 = 3 / u8->f32 = 4 / i8 = 5 / u8->i8 = 6> <outputFormatToggle (pkd->pkd = 0 / pkd->pln = 1)> <case number = 0:86> <verbosity = 0/1>\n");
```

Comment:
since now all layouts are being run using a single script
- Remove the print statements in Line 144, 146, 149 and combine them into single print statement 
- Remove the usage of Tensor_host_pkd3 / Tensor_host_pln3 / Tensor_host_pln1